examples/plot_timeseries_example.py

```diff
@@ -0,0 +1,117 @@
+"""
+=======================================================
+Estimating prediction intervals of time series forecast
+=======================================================
+This example uses :class:`mapie.estimators.MapieRegressor` to estimate
+prediction intervals associated with time series forecast. We use the
+standard cross-validation approach to estimate residuals and associated
+prediction intervals.
+
+We use here the Victoria electricity demand dataset used in the book
+"Forecasting: Principles and Practice" by R. J. Hyndman and G. Athanasopoulos.
+The electricity demand features daily and weekly seasonalities and is impacted
+by the temperature, considered here as a exogeneous variable.
+
+The data is modelled by a Random Forest model with a
+:class:`sklearn.model_selection.RandomizedSearchCV` using a sequential
+`sklearn.model_selection.TimeSeriesSplit` cross validation, in which the
+training set is prior to the validation set.
+The best model is then feeded into :class:`mapie.estimators.MapieRegressor`
+to estimate the associated prediction intervals.
+We consider the standard CV+ resampling method.
```

Comment:
Ajouter un paragraphe "known limitations", qui précise qu'en général, il n'y a plus de garanties théoriques pour le TS, mais que en l'occurrence, ça marche bien empiriquement notamment car il n'y a pas de tendance manifest

Reply:
J'ai ajouté le paragraphe ci-dessous. 

Je ne vois qu'une seule grosse limite à l'utilisation d'un `KFold` classique sur des données TS, le fait qu'on utilise des données d'entraînement ultérieures aux données de validation pour estimer les résidus, ce qui est normalement incompatible avec l'ordre chronologique des données. Cependant, je pense que les garanties mathématiques restent valables avec cet objet, même avec une TS avec tendance. Les données de test deviendraient alors "out-of-distribution" mais l'erreur épistémique augmenterait.   

Comme discuté avec Nicolas, l'utilisation d'un `TimeSeriesSplit` est plus pertinente pour ce type de données mais on casserait en l'occurrence les garanties mathématiques, a fortiori sur des données avec tendance. 

```
We wish to emphasize one main limitation with this example. 
We use a standard cross-validation in Mapie to estimate the prediction intervals,
through the `sklearn.model_selection.KFold()` object. 
Residuals are therefore estimated using models trained on data with higher indices
than the validation data, which is inappropriate for time-series data. 
Howerver, using a `sklearn.model_selection.TimeSeriesSplit` cross validation object for
estimating the residuals breaks the theoretical guarantees of the Jackknife+ and CV+
methods.
```